### DISCUSSION TITLE
bugs with external s3

### DISCUSSION THREAD

**Discussion by lavender2020** [milvus-io/milvus#20825] [Q&A and General discussion]
<img width="1791" alt="image" src="https://user-images.githubusercontent.com/4230433/203836977-0d73bd14-1439-49d0-86ba-b0cf793ecdd6.png">

**Comment by lavender2020** [comment]
Client Version: v1.25.0
Server Version: v1.21.14-eks-fb459a0
Milvus helm repo: master branch

Values.yaml

<img width="731" alt="image" src="https://user-images.githubusercontent.com/4230433/203837546-c0466439-9b60-4afa-8ddf-3fe7a8c55f28.png">

**Reply by LoveEachDay** [reply]
@lavender2020 Could you check the config under `minio` section in the `configs/milvus.yaml` ? Seems the `minio.address` is messed up with the s3 address and local minio address at the same time.

Do you manually change the helm charts before you install milvus cluster?
